e2e/install_test.go

```diff
@@ -996,7 +996,7 @@ func TestSingleNodeAirgapUpgradeCustomCIDR(t *testing.T) {
 	// expected to be in the 192.168.0.0/17 range while services are in the
 	// 192.168.128.0/17 range. i.e. pods are in 192.168.0-127.* while
 	// services in 192.168.128-254.*.
-	podregex := `192\.168\.\([0-9]\|[1-9][0-9]\|12[0-7]\)\.`
+	podregex := `192\.168\.\([0-9]\|[1-9][0-9]\|12[0-7]\|1[0-1][0-9]\)\.`
```

Comment:
Can we somehow simplify this ?

Reply:
my suggestion for simplifying this is "use a /15 so that each sub range is a /16, and you only have to match the first two numbers in the quad"